reproduce.sh

```diff
@@ -32,12 +32,14 @@ while [ "$1" != "" ]; do
 done
 
 mkdir -p $CLUSTERFUZZ_CONFIG_DIR
-if [ ! -d $ROOT_DIRECTORY/ENV ] || ([ $additional_deps_args ] && [ ! -d $ROOT_DIRECTORY/local/bin/android-sdk ]); then
+installed_version=$(cat $CLUSTERFUZZ_CONFIG_DIR/version 2>/dev/null || echo -n)
```

Comment:
This should check file exists in an if rather than 2>&1 stuff

Reply:
Done